packages/bsky/package.json

```diff
@@ -8,9 +8,8 @@
     "codegen": "lex gen-server ./src/lexicon ../../lexicons/com/atproto/*/* ../../lexicons/app/bsky/*/*",
```

Comment:
yup might not even need to codegen atproto schemas 👀

Reply:
Oooh that would be nice.  Right now I think we only use one or two of the sync lexicons.  Maybe we can special-case those—I will take a look at it once non-repo updates settles!